gapii/cc/spy.cpp

```diff
@@ -262,7 +263,15 @@ std::shared_ptr<StaticContextState> GlesSpy::GetEGLStaticContextState(CallObserv
 
     Constants constants;
     getContextConstants(constants);
-    std::shared_ptr<StaticContextState> out(new StaticContextState(constants));
+
+    std::string threadName;
+#if TARGET_OS == GAPID_OS_ANDROID
+    char buffer[256] = { 0 };
```

Comment:
Ubernit: 256 seems a little excessive, [this documentation](http://man7.org/linux/man-pages/man2/prctl.2.html) claims the string is no longer than 15 characters + null-terminator